packages/scoped-elements/test-web/components/vanilla/MyButton3.js

```diff
@@ -0,0 +1,13 @@
+import { LitElement } from 'lit';
+import { ScopedElementsMixin } from '../../../html-element.js';
+
+export class MyButton3 extends ScopedElementsMixin(LitElement) {
```

Comment:
Can we have this extend from HTMLElement instead? Nice to keep things as minimal as possible